README.md

```diff
@@ -39,6 +39,36 @@ sulu_headless:
     resource: "@SuluHeadlessBundle/Resources/config/routing_website.yml"
 ```
 
+## Usage
+
+That you can provide data of your pages using the headless api you need to use the following controller in your pages templates:
+
+### config/templates/pages/
+
+```xml
+<view>pages/headless</view>
+<controller>Sulu\Bundle\HeadlessBundle\Controller\HeadlessWebsiteController::indexAction</controller>
+```
+
+### templates/pages/headless.html.twig
+
+If you use the provided setup you should have atleast the following on your `headless.html.twig` template:
```

Comment:
If you want to use the frontend setup included in this bundle, you should include the following lines in your `headless.html.twig` template to setup and start the react application: